test.py

```diff
@@ -0,0 +1,94 @@
+#fasta
+fasta = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
```

Comment:
`"S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta"`というふうに文字列使ってもいいけど、Pathであることが明示的である方が親切
`pathlib.Path`を使おう
```suggestion
from pathlib import Path
fasta = Path("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta").open()
```